kinoml/features/kinase.py

```diff
@@ -0,0 +1,61 @@
+"""
+kinase_model.py
+Defines the Kinase class
+
+"""
+
+class Kinase(object):
```

Comment:
This needs to be a subclass of `Protein` or `Structure`, with specialized features. It should also live somewhere else, not in `kinoml.features`.